### DISCUSSION TITLE
Release v3.22.0

### DISCUSSION THREAD

**Discussion by fonnebien** [serverless-operations/serverless-step-functions#624] [General]
Hello!

 I was wondering when we can expect a new release of the `master` branch as `v3.22.0`? With the new serverless version, I think that it's necessary to not be blocking other people who depend on this package in their project(s). 

Thank you in advance for your help and the effort put in this library.

@horike37 @lopburny @theburningmonk

**Comment by zirkelc** [comment]
We released v3.23.0 a few days ago, please check if it resolved any impediments for you

**Reply by fonnebien** [reply]
The issue was already resolved in v3.22.0. Thank you for taking your time to come back to this discussion!
